crt_portal/static/sass/custom/progress.scss

```diff
@@ -67,18 +67,20 @@ ol.steps {
 
     a {
       display: block;
+      margin-top: -($circle-diameter + 6.5px); // 6.5px == size of 0.5em bottom margin on step circle
+      padding-top: $circle-diameter + 6.5px;
     }
   }
 
-  .step a::before,
-  .step a::after {
```

Comment:
okay, this was a bad idea when I did it before because it only numbers anchor links, and the public side doesn't have any 🙈

Reply:
It seemed like a great idea at the time. We were younger then...